commit ef17fdacc03db41508f970710132d1036f891d47 (refs/changes/62/5462/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-02-03T11:26:37+00:00 (8 months ago)
    
    MANTA-4093 nginx should not modify its source tree during the build
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/deps/nginx b/deps/nginx
index f054a7d..4d88576 160000
--- a/deps/nginx
+++ b/deps/nginx
@@ -1 +1 @@
-Subproject commit f054a7df29b99932cbe50c1216d545029b2e769c
+Subproject commit 4d885764f41be161c907cb8049d79cf0d6558981
